tensorboard/compat/proto/BUILD

```diff
@@ -48,10 +48,6 @@ py_test(
     size = "small",
     srcs = ["proto_test.py"],
     srcs_version = "PY2AND3",
-    tags = [
-        "manual",
```

Comment:
I've actually already done part 2 above since we needed it for our 1.13 release: https://github.com/tensorflow/tensorboard/pull/1823

So those changes can just be dropped from this PR.